tests/TestCase/Database/Schema/MysqlSchemaTest.php

```diff
@@ -1229,8 +1229,9 @@ public function testDescribeJson()
     protected function _getMockedDriver()
     {
         $driver = new \Cake\Database\Driver\Mysql();
-        $mock = $this->getMockBuilder('FakePdo')
+        $mock = $this->getMockBuilder(\PDO::class)
```

Comment:
@dereuromark I'll update that later and try to improve the use of `::class` where it's possible.
I've no time left today and will be alreday happy if the testsuite is working again 😉